src/main/io/displayport_msp_bf_compat.c

```diff
@@ -328,28 +328,28 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 
     case SYM_ZERO_HALF_LEADING_DOT:
         return BF_SYM_ZERO_HALF_LEADING_DOT;
+*/
 
     case SYM_AUTO_THR0:
-        return BF_SYM_AUTO_THR0;
+        return 'A';
 
     case SYM_AUTO_THR1:
-        return BF_SYM_AUTO_THR1;
+        return 'T';
 
     case SYM_ROLL_LEFT:
-        return BF_SYM_ROLL_LEFT;
+        return BF_SYM_ROLL;
 
     case SYM_ROLL_LEVEL:
-        return BF_SYM_ROLL_LEVEL;
+        return BF_SYM_ROLL;
 
     case SYM_ROLL_RIGHT:
-        return BF_SYM_ROLL_RIGHT;
+        return BF_SYM_ROLL;
 
     case SYM_PITCH_UP:
-        return BF_SYM_PITCH_UP;
+        return BF_SYM_PITCH;
 
     case SYM_PITCH_DOWN:
-        return BF_SYM_PITCH_DOWN;
- */
+        return BF_SYM_PITCH;
```

Comment:
Same comment as roll. Ambiguity is not removed. You have no was of knowing if it's pitching up or down.

Reply:
Same thing as roll. It tells you how many degrees of pitch there is. Say you want a 15 degree climb angle. Just pitch up until the number hits 15. As with the roll, just looking at the horizon will tell you if you are pitching up or down. There is no ambiguity.